eng/depProj.targets

```diff
@@ -47,6 +61,18 @@ See the LICENSE file in the project root for more information.
     <CopyNuGetImplementations>true</CopyNuGetImplementations>
   </PropertyGroup>
 
+  <Target Name="RemoveImplicitPackageReferences">
```

Comment:
Are there references beyond what arcade is adding? Are you trying to get the ones from the sdk as well? If it is just arcade then we can use https://github.com/dotnet/arcade/pull/1240 to clean that up.

Reply:
Yeah, it was just the ones arcade was adding (previous property turns off all from SDK).